projects/demo/src/assets/snippets/material.css.snippet.css

```diff
@@ -0,0 +1,3 @@
+val-errors div {
```

Comment:
You used ng-deep in the demo but not here. Normal?

Reply:
Yes. This shows how to do it in the global stylesheets (because you normally use material in the whole app). But in our demo, where material is used only in one component, I chose to set the rule inside the component in order to avoid impacting all the other components